tests/test_repositories_seeding.py

```diff
@@ -113,17 +114,19 @@ def get_container(
         if tmp_path
         else {}
     )
+    container = None
     try:
         container = client.containers.run(
-            image="t4c/client/remote",
+            image=os.getenv("DOCKER_CAPELLA_READONLY", "t4c/client/remote"),
```

Comment:
This is interfering. It's not the read-only image.

Reply:
fixed